pyproject.toml

```diff
@@ -9,3 +9,7 @@ ignore-words-list = "inout,numer,startd,couldn,referr"
 builtin = "clear"
 # disable warnings about binary files and wrong encoding
 quiet-level = 3
+
+[tool.cython-lint]
+max-line-length = 999
```

Comment:
I'd like to re-enable E501 with a reasonable line length limit (following our flake8/black conventions) in a follow-up PR. Wrapping lines takes a little more manual effort than I wanted to put in for this first pass.